### PR TITLE
Reorder example solves and add some juicy new example solves

### PR DESCRIPTION
**Pull request by stanley-su**

### CODE REVIEW

File: src/fb.md [hunk 1]
```diff
@@ -246,13 +243,18 @@ after the first stage.
...
-`L' B2` builds the bottom line of FB and orients the blue-red edge.
+`F' D` builds the front square. 
+Back pair's corner is clearly uninfluencable, so predict the back pair's corner's position and orientation, and then aim to influence the back pair's edge. 
 
-Starting with `B'` leverages the `B2` to also insert the back edge, for a `line + line` block.
+We can predict the back pair's corner will be at DBR after first stage, with its D-face sticker on the bottom. 
+
+We should know from experience that the back pair's edge needs to be misoriented in BR to pair with the back pair's corner, when the back pair's corner is in DBR with its D-face sticker on the bottom.
+
+Thus, starting with `R` setups up a connected last pair for an easy 3 move second stage.
```
Comment (Rouxles) [inline src/fb.md:257]: i think saying last pair is probably clearer here

Comment (Rouxles) [reply src/fb.md:257]: or perhaps even fb last pair, idk

File: src/fb.md [hunk 2]
```diff
@@ -246,13 +243,18 @@ after the first stage.
...
 </script>
 
-`L' B2` builds the bottom line of FB and orients the blue-red edge.
+`F' D` builds the front square. 
+Back pair's corner is clearly uninfluencable, so predict the back pair's corner's position and orientation, and then aim to influence the back pair's edge. 
 
-Starting with `B'` leverages the `B2` to also insert the back edge, for a `line + line` block.
+We can predict the back pair's corner will be at DBR after first stage, with its D-face sticker on the bottom. 
+
+We should know from experience that the back pair's edge needs to be misoriented in BR to pair with the back pair's corner, when the back pair's corner is in DBR with its D-face sticker on the bottom.
```
Comment (Rouxles) [inline src/fb.md:255]: remove the comma?

File: src/fb.md (hunk 1 above)
Comment (stanley-su) [reply src/fb.md:257]: I disagree; it hints towards the location of the pair, providing more information than just 'last pair'. 'Front pair' has also been used multiple times in this document.
'Fb last pair' is verbose, as this is the FB document.

File: src/fb.md (hunk 2 above)
Comment (stanley-su) [reply src/fb.md]: Yes, thanks. Was worried about sentence getting too long, but it's better now.

**Review by Rouxles — APPROVED** [review]

**Pull request merged**